packages/blade/src/components/Modal/docs/ModalExamples.stories.tsx

```diff
@@ -0,0 +1,72 @@
+import React from 'react';
+import type { ComponentStory, Meta } from '@storybook/react';
+import { Modal } from '../Modal';
+import {
+  BasicModalStory,
+  ModalStackingStory,
+  ModalWithHeaderFooterStory,
+  ModalWithScrollableBackgroundStory,
+  ModalWithScrollableContentStory,
+} from './stories';
+import { Sandbox } from '~src/_helpers/storybook/Sandbox';
+
+const ModalMeta: Meta = {
+  title: 'Components/Modal/Examples',
+  component: Modal,
+  parameters: {
+    viewMode: 'story',
+    options: {
+      showPanel: false,
+    },
+    previewTabs: {
+      'storybook/docs/panel': {
+        hidden: true,
+      },
+    },
+    chromatic: { disableSnapshot: true },
```

Comment:
Why are we disabling snapshots? We should do visual tests.

Reply:
Added one separately